components/pkg-export-kubernetes/src/cli.rs

```diff
@@ -107,34 +119,32 @@ impl<'a, 'b> Cli<'a, 'b> {
                             "group is a logical grouping of services with the same package and \
                          topology type connected together in a ring (default: default)",
                         ),
+                )
+                .arg(
+                    Arg::with_name("CONFIG")
+                        .value_name("CONFIG")
+                        .long("config")
+                        .short("n")
+                        .help(
+                            "The path to habitat configuration file in user.toml format. Habitat \
+                            will use it for initial configuration of the service",
```

Comment:
of the service running inside of Kubernetes.

Reply:
well that's clear from the context I'd say but I'll add it since you prefer. :)